Classes/Public/Metrics.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
```

Comment:
Maybe move this file to the `Constants` subfolder (the one that's in the same directory as this file).